source/fundamentals/enterprise-authentication.txt

```diff
@@ -2,4 +2,261 @@
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version of
+{+mongo-enterprise+}.
+
+- :ref:`csharp-kerberos`
+- :ref:`csharp-LDAP`
+
+To authenticate using another mechanism, see the
+:ref:`Authentication Mechanisms guide <csharp-authentication-mechanisms>`. For
+more information on establishing a connection to your MongoDB cluster, see the
+:ref:`csharp-connect-to-mongodb`.
+
+Specify an Authentication Mechanism
+-----------------------------------
+
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using either of the following methods:
+
+- A **connection string**, also known as a **connection URI**, which is a string
+  that tells the driver how to connect to a MongoDB deployment and how to behave while
+  connected.
+
+- A factory method for the supported authentication mechanism, contained in the 
+  ``MongoCredential`` class.
+.. _csharp-kerberos:
+
+GSSAPI/Kerberos
+---------------
+
+The Generic Security Services API (``GSSAPI``) authentication mechanism allows the 
+user to authenticate to a Kerberos service using the user's principal name.
+
+The following examples specify the authentication mechanism using the following
+placeholders:
+
+- ``<username>``: Your URL-encoded principal name, e.g. "username%40REALM.ME"
+- ``<password>``: Your Kerberos user's password
+- ``<hostname>``: The network address of your MongoDB server, accessible by your client
+
+Select the :guilabel:`Connection String` or :guilabel:`MongoCredential` tab to
+see the corresponding syntax for specifying the ``GSSAPI/Kerberos`` authentication
+mechanism:
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connection-string
+
+      .. code-block:: csharp
+
+         var mongoClient = new MongoClient("mongodb://<username>:<password>@<hostname>/?authMechanism=GSSAPI");
+
+   .. tab:: MongoCredential
+      :tabid: mongo-credential
+
+      .. code-block:: csharp
+         :emphasize-lines: 1
+
+         var credential = MongoCredential.CreateGssapiCredential("<username>", "<password>");
+         var settings = MongoClientSettings.FromConnectionString("<connection string>");
+         settings.Credential = credentials;
+         var mongoClient = new MongoClient(settings);
+
+.. tip:: Omit Password
+
+   You can omit the password if one of the following are true:
+
+   - On Windows, the process owner running the application is the same as the
+     user needing authentication.
+
+   - ON Linux, the user has initialized their keytab via ``kinit username@REALM.COM``.
+  
+Additional Properties
+~~~~~~~~~~~~~~~~~~~~~
+
+You can specify additional properties to your authentication mechanism using a
+connection string, or a factory method in the ``MongoCredential`` class.
+
+- Fully qualified domain name
```

Comment:
Issue: The list elements kind of get lost with all the text and code samples.

Suggestion: Change to third-level headings (or something else?)

Reply:
Agreed, I think the style guide mentions trying to avoid 3rd level headings, but I also think that's the best option in this case.